themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,106 @@
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Earlier this year, Pulumi's user experience (UX) team sat down with users to understand how we can improve the Pulumi experience. It's important not only to the UX team, but across the company, that we listen to the real wants, needs, pains, and excitements of the people who use our product.
+
+The [Pulumi Service](https://www.pulumi.com/docs/intro/pulumi-service/) was at the center of many conversations we had with users. The Pulumi Service is the easiest way to use Pulumi’s open source, universal infrastructure as code SDK at scale and provides a fully managed experience. The team is always exploring how we can deliver the best SaaS experience we can. But having input from the community gave us a meaningful point of entry.
+
+<!--more-->
+
+Here are the two problems our users faced that we wanted to improve:
+
+1. Users were losing time trying to get where they needed to go. Users need to find important content faster and more easily.
+2. Users didn't feel like they understood the "big picture" of their Pulumi usage. We wanted to offer improved line of sight into organizations' day-to-day operations.
+
+We knew we could do better. Now, we're excited to share the changes we've made so far!
+
+## New primary navigation pane
+
+Next time you sign in to the Pulumi Service, look for the navigation pane on the left hand side of the page. All primary navigation items like your Dashboard, Stacks, and Settings are accessible in the new pane from anywhere in the Service.
+
+![""](dashboard-nav-expanded.png)
+
+The navigation pane consolidates actions and information so it's easy for you to find common items. You'll find options in the pane to:
+
+- switch between your individual account and any other organizations you're part of
+- create new organizations
+- access your favorite stacks (read on to find out more!)
+- directly access sub-pages like Billing or Access Tokens
+- send us your feedback!
+
+![""](dashboard-nav-collapsed.png)
+
+We made these changes to help you move through the Pulumi Service without losing your place and ensure convenient access to key pages. Even if you collapse the navigation pane for more screen real estate, you can still get where you need to go!
+
+If you're looking for your account details, they have moved into the black utility toolbar. To access your account settings or logout of the Pulumi Service, select your profile image at the top right of your browser window.
+
+## Stack Favorites
+
+You can now pin your favorite stacks, so you can quickly access the stacks you need most. This reduces your time spent combing through projects. Look for the star icon next to any stack name and select it to start building out your favorites list.
```

Comment:
Might be nice to have a short intro here - along the lines of:

"Most Pulumi users have access to dozens of hundreds of stacks, but regularly interact with a few particularly important stacks as part of their daily usage.  We've introduced a new Stack Favorites feature to let users identify the stacks that are most important to them, and to offer easy access to those stacks from anywhere in the console"